src/building-blocks/openTelemetry/open-telemetry.module.ts

```diff
@@ -6,7 +6,7 @@ import { OpenTelemetryOptions, OpenTelemetryTracer} from './open-telemetry-trace
     provide: 'IOpenTelemetryTracer',
     useClass: OpenTelemetryTracer
   }],
-  exports: ['IOpenTelemetryTracer']
+  exports: ['IOpenTelemetryTracer', OpenTelemetryOptions]
```

Comment:
Why did you expose `OpenTelemetryOptions` here? we don't need to expose `OpenTelemetryOptions` to outside. we only need `IOpenTelemetryTracer` to use tracer.